### PR TITLE
utils.rs - method, authority and path

### PR DESCRIPTION
**Pull request by tzuryby**
utils.rs - method, authority and path

### CODE REVIEW

File: curiefense/curieproxy/rust/curiefense/src/utils.rs [hunk 1]
```diff
@@ -211,7 +211,11 @@ impl RequestInfo {
                 "path": self.rinfo.qinfo.qpath,
                 "query": self.rinfo.qinfo.query,
                 "ip": self.rinfo.geoip.ipstr,
-                "ipnum": ipnum
+                "ipnum": ipnum,
+                "authority": self.rinfo.meta.authority,
+                "method": self.rinfo.meta.method,
+                "path": self.rinfo.meta.path //,
+                // "extra": self.rinfo.meta.extra
```
Comment (flaper87) [inline curiefense/curieproxy/rust/curiefense/src/utils.rs:218]: can we remove this line here? Doesn't look it's needed/used

**Review by flaper87 — CHANGES_REQUESTED** [review]
@tzuryby do you think you can split this PR in 2? One for the changes in `utils.rs` and the other one for `waf.rs`. These look like unrelated changes so I would like to tackle the changes separately.

The change to `utils.rs` will solve #410 as far as I can tell.

File: curiefense/curieproxy/rust/curiefense/src/waf.rs [hunk 2]
```diff
@@ -62,49 +62,56 @@ impl WafBlock {
...
-                "value": wmatch.value,
-                "value": {
-                    "msg": "SQLi",
-                    "fingerprint": fp
-                }
+                "value": wmatch.value
+                // "value": {
+                //     "msg": "SQLi",
+                //     "fingerprint": _fp
+                // }
```
Comment (flaper87) [inline curiefense/curieproxy/rust/curiefense/src/waf.rs:88]: It'd be nice to remove all the commented lines before merging this PR :)

File: curiefense/curieproxy/rust/Dockerfile [hunk 3]
```diff
@@ -9,10 +9,12 @@ RUN apt-get update && \
...
-RUN luarocks install lua-cjson && \
-    luarocks install luafilesystem && \
-    luarocks install luasocket && \
-    luarocks install redis-lua
+#RUN luarocks install lua-cjson && \
+#    luarocks install luafilesystem && \
+#    luarocks install luasocket && \
+#    luarocks install redis-lua
+
+RUN luarocks install lua-cjson
```
Comment (flaper87) [inline curiefense/curieproxy/rust/Dockerfile:17]: Is this just a test?

**Review by flaper87 — CHANGES_REQUESTED** [review]

**Comment by bartavelle** [comment]
Of course I did not realize there already was a pull request for this! I just proposed #442 .

**Comment by flaper87** [comment]
> Of course I did not realize there already was a pull request for this! I just proposed #442 .

It's ok! This one is doing a bit more than it should 😂

**Comment by tzuryby** [comment]
@bartavelle merged those in his code already.

**Pull request closed without merging**